### PR TITLE
updated Gemfile

### PR DESCRIPTION
**Pull request by MelsenC**
updated Gemfile to remove gems that result in you having to restart your server any time you edit a model file. Now we should only have to restart the server when we want to edit config files. Also installed the pry-rails gem for debugging.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -67,5 +63,5 @@ gem 'simple_form'
 
 gem 'devise'
 
-
+gem 'pry-rails'
```
Comment (mscoutermarsh) [inline Gemfile]: then can go inside group development & test. No need to have it load in production.

**Comment by mscoutermarsh** [comment]
I'm not familiar with `Listen` - maybe new in Rails 5?

Seems very strange it requires you to restart the server.

Is it not picking up changes in the Pawn/King models?

Does `Piece` work?

**Comment by MelsenC** [comment]
Piece does seem to work. I just was restarting my server every time I edited piece.rb, or game.rb, and my mentor Colin said these are issues that are bothering lots of rails developers right now.

File: Gemfile (hunk 1 above)
Comment (MelsenC) [inline Gemfile]: oh good point! Will adjust tonight..

**Comment by mscoutermarsh** [comment]
Strange. I'd search the Rails issues on github, see if you can find any info.

Usually, keeping Spring is helpful (makes startup, running a console, running tests, WAY faster).

**Comment by MelsenC** [comment]
I couldn't find enough evidence online to support getting rid of those gems. I'm going to uncomment config.file_watcher = ActiveSupport::EventedFileUpdateChecker in the development.rb file too. This pull request will just turn into a request to install pry-rails.

**Pull request merged**